.github/workflows/compile-check.yml

```
@@ -0,0 +1,26 @@
+name: compile-check
+
+on: [push, pull_request]
+
+jobs:
+  compile:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          path: "mods"
+      
+      - name: Compile Scripts
+        uses: ASpoonPlaysGames/squirrel-re-compiler@v1
+        with:
+          mods-directory: "${{ github.workspace }}/mods"
+
+      - name: Remove Northstar.Custom
+        run: rmdir ${{ github.workspace }}\mods\Northstar.Custom /s /q
+        shell: cmd
+          
+      - name: Compile Scripts (No Northstar.Custom)
+        uses: ASpoonPlaysGames/squirrel-re-compiler@main
```

```suggestion
        uses: ASpoonPlaysGames/squirrel-re-compiler@v1
```
for some reason i forgot to change this one as well, or github did a weird thing idk tbh